### ISSUE TITLE
JSON pipeline reader doesn't propagate CLI specified --verbose and --debug args

### ISSUE THREAD

**Issue by chambbj** [PDAL/PDAL#1208]
I just broke this in the original implementation. Per stage specified values still work fine.

**Comment by chambbj** [comment]
Closed via #1209

**Comment by chambbj** [comment]
This appears to have reopened with c29f692827ba192ffe398524b610eea9e3d90750.

```
bin/pdal pipeline -i test/data/pipeline/attribute.json -v 10 --debug
```

shows no output.

Prior to this commit, I got

```
$ bin/pdal pipeline -i test/data/pipeline/attribute.json -v 10 --debug
(filters.attribute Debug: 3): GDAL debug: GDAL: GDALOpen(/Users/chambbj/loki/pdal/repo/test/data/autzen/attributes.shp, this=0x7f9ce152d140) succeeds as ESRI Shapefile.
(filters.attribute Debug: 3): GDAL debug: GDAL: GDALOpen(/Users/chambbj/loki/pdal/repo/test/data/autzen/attributes.shp, this=0x7f9ce1566b50) succeeds as ESRI Shapefile.
(writers.las Debug: 3): Wrote 1065 points to the LAS file
(writers.las Debug: 3): GDAL debug: Shape: 5 features read on layer 'attributes'.
(writers.las Debug: 3): GDAL debug: GDAL: GDALClose(/Users/chambbj/loki/pdal/repo/test/data/autzen/attributes.shp, this=0x7f9ce1566b50)
(writers.las Debug: 3): GDAL debug: Shape: 5 features read on layer 'attributes'.
(writers.las Debug: 3): GDAL debug: GDAL: GDALClose(/Users/chambbj/loki/pdal/repo/test/data/autzen/attributes.shp, this=0x7f9ce152d140)
(writers.las Debug: 3): GDAL debug: GDAL: In GDALDestroy - unloading GDAL shared library.
```
